_sass/layouts.scss

```diff
@@ -0,0 +1,81 @@
+.open-covid-listing ul {
```

Comment:
Be sure to change the class names to match the repo name "pandemic-response-repository". If these rely on autogenerated class names, they won't catch.